Added Kinect rules and fixed the owner in the PrimeSense rules.

From: Roman Donchenko <roman.donchenko@itseez.com>

The Kinect rules are from <http://github.com/avin2/SensorKinect/>.
---
 Platform/Linux/Install/55-primesense-usb.rules |   19 +++++++++++++------
 1 files changed, 13 insertions(+), 6 deletions(-)

diff --git a/Platform/Linux/Install/55-primesense-usb.rules b/Platform/Linux/Install/55-primesense-usb.rules
index 6294f40..621fe3b 100644
--- a/Platform/Linux/Install/55-primesense-usb.rules
+++ b/Platform/Linux/Install/55-primesense-usb.rules
@@ -1,8 +1,15 @@
 # make primesense device mount with writing permissions (default is read only for unknown devices)
-SYSFS{idProduct}=="0200",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0300",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0400",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0500",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0600",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0601",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
+SYSFS{idProduct}=="0200",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0300",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0400",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0500",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0600",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0601",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
 
+#--avin mod--
+# Kinect
+SUBSYSTEM=="usb", ATTR{idProduct}=="02ae", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="video"
+SUBSYSTEM=="usb", ATTR{idProduct}=="02ad", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="audio"
+SUBSYSTEM=="usb", ATTR{idProduct}=="02b0", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="video"
+SUBSYSTEM=="usb", ATTR{idProduct}=="02be", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="audio"
+SUBSYSTEM=="usb", ATTR{idProduct}=="02bf", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="video"
